docs/binary-tarball-install.md

```diff
@@ -4,10 +4,20 @@ A binary tarball contains a group of files, including the source code, bundled t
 
 See the list of the [binary tarball available based on the Percona Server for MySQL version](binary-tarball-names.md) to select the right tarball for your environment.
     
-You can download the binary tarballs from the `Linux - Generic` [section](https://www.percona.com/downloads/Percona-Server-8.0/LATEST/binary/tarball/) on the download page.
+You can download the binary tarballs from the `Linux - Generic` [section](https://www.percona.com/downloads) on the download page.
 
-Fetch and extract the correct binary tarball. For example for *Debian 10*:
+Fetch and extract the correct binary tarball. For example, for Debian 10:
 
 ```{.bash data-prompt="$"}
 $ wget https://downloads.percona.com/downloads/Percona-Server-8.0/Percona-Server-8.0.26-16/binary/tarball/Percona-Server-8.0.26-16-Linux.x86_64.glibc2.12.tar.gz
 ```
+
+## Install Percona Server for MySQL Pro from a binary tarball
+
+You can download the required binary tarball for Percona Server for MySQL Pro using your `CLIENTID` and `TOKEN` in the following link https://repo.percona.com/private/[CLIENTID]-[TOKEN]/ps-80-pro/.
+
+Fetch and extract the correct binary tarball using your `CLIENTID` and `TOKEN`. For example, for Debian 12:
```

Comment:
Let's better give an example for ol9 as we announce that Deb12 ad Jammy are more for testing